web/massastation/src/pages/store/MyPlugin.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+
+import {
+  Button,
+  Certificate,
+  MassaWallet,
+  Plugin,
+} from '@massalabs/react-ui-kit';
+import { FiArrowUpRight, FiRefreshCcw, FiTrash2 } from 'react-icons/fi';
+import { IMassaPlugin } from './MyStation';
+import { massalabsNomination } from '../../utils/nominationStandards';
+import { existsInArray } from '../../utils/utils';
+
+enum PluginStatus {
+  // Up And Down are sent by the BE, we use the On/Off standard to stick to design
+  On = 'Up',
+  Off = 'Down',
```

Comment:
```suggestion
  On = 'On',
  Off = 'Off',
```

Reply:
Forget that if in BE is up/down.

Sorry!

Reply:
we can't do that because the backend expects 'Up'/'Down" is that correct @Pierregvx ?